packages/related-keyphrase-suggestions/src/elements/IntentBadge/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useState } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import { Tooltip } from "@yoast/ui-library";
+import { __ } from "@wordpress/i18n";
+
+const variants = {
+	i: {
+		title: __( "Informational", "wordpress-seo" ),
+		description: __( "The user wants to find an answer to a specific question.", "wordpress-seo" ),
+	},
+	n: {
+		title: __( "Navigational", "wordpress-seo" ),
+		description: __( "The user wants to find a specific page or site.", "wordpress-seo" ),
+	},
+	c: {
+		title: __( "Commercial", "wordpress-seo" ),
+		description: __( "The user wants to investigate brands or services.", "wordpress-seo" ),
+	},
+	t: {
+		title: __( "Transactional", "wordpress-seo" ),
+		description: __( "The user wants to complete an action (conversion).", "wordpress-seo" ),
+	},
+};
+
+/**
+ *
+ * @param {string} initial The initial of the intent.
+ *
+ * @returns {JSX.Element} The colored initial badge.
+ */
+const IntentBadge = ( { initial } ) => {
+	const [ isVisible, setIsVisible ] = useState( false );
+	const handleMouseEnter = useCallback(
+		() => setIsVisible( true ),
+		[ setIsVisible ],
+	);
+	const handleMouseLeave = useCallback(
+		() => setIsVisible( false ),
+		[ setIsVisible ],
+	);
+
+
+	if ( ! variants[ initial ] ) {
+		return null;
+	}
+
+	return (
+		<div
+			aria-label={ `${ variants[ initial ].title }, ${ variants[ initial ].description }` }
```

Comment:
you can reach the tooltip via tab since it's mouse in and mouse out. And if it was possible, then there war no need for aria-label, the screen reader would read the content of the tooltip.